src/memory.jl

```diff
@@ -330,10 +330,13 @@ end
 function unsafe_copy3d!(dst::Union{Ptr{T},CuPtr{T}}, dstTyp::Type{<:Buffer},
                         src::Union{Ptr{T},CuPtr{T}}, srcTyp::Type{<:Buffer},
                         width::Integer, height::Integer=1, depth::Integer=1;
-                        srcPos::CuDim3=CuDim3(0,0,0), dstPos::CuDim3=CuDim3(0,0,0),
+                        srcPos::CuDim=(1,1,1), dstPos::CuDim=(1,1,1),
```

Comment:
Shouln't the default value be CuDim(1,1,1) instead of (1,1,1)?

Reply:
CuDim is just an alias for either an integer, or a tuple or 1, 2 or 3 items.